### ISSUE TITLE
Punctuation characters are not announced correctly when used in aria-keyshortcuts

### ISSUE THREAD

**Issue by stoyanovski** [FreedomScientific/VFO-standards-support#653]
### Summary

> The issue is reproducible in the following example: https://jsbin.com/cizayaheyi/edit?html,output
> Start JAWS and set the focus on the button

### Expected result

> The keyboard shortcut should be announced as "Control plus comma"

### Actual result

> The following is announced: 
> "Settings **Control plus** Button To activate press Enter **Control plus comma**"

### Example

> https://jsbin.com/cizayaheyi/edit?html,output

### Additional Information

> WAI-ARIA Specification for aria-keyshortcuts: https://rawgit.com/w3c/aria/master/#aria-keyshortcuts
> Examples of valid keyboard shortcuts include:
> - "A"
> - "Shift+Space"
> - "Control+Alt+." (the dot is not announced either)
> - "Control+Shift+&#39;"
> - "Alt+Shift+P Control+F"
> - "Meta+C Meta+Shift+C"

#### JAWS version and build number

> JAWS 2022.22.06.9 ILM
 
#### Operating System and version

> Windows 10

#### Browser and version:

> Chrome 103.0.5060.114
